packages/rspack-cli/src/commands/build.ts

```diff
@@ -21,6 +21,15 @@ export class BuildCommand implements RspackCommand {
 					}
 				}),
 			async options => {
+				if (options.watch) {
+					// @ts-expect-error
+					setBuiltinEnvArg(options.env, "WATCH", true);
```

Comment:
Does webpack support this?

Reply:
> Does webpack support this?

yes webpack-cli support it.

https://github.com/webpack/webpack-cli/blob/eea6adf7d34dfbfd3b5b784ece4a4664834f5a6a/packages/webpack-cli/src/webpack-cli.ts#L2500-L2503

WEBPACK_BUILD / WEBPACK_BUNDLE for non-watch build
WEBPACK_WATCH for watch build
WEBPACK_SERVE for serve

thus i add `RSPACK_*` correspondingly.

another question is whether we should add these `WEBPACK_*` together to align with webpack-cli. I didn't do it as it will break those who use `WEBPACK_*` to determine who (webpack or rspack) is using the function configuration .

Reply:
> another question is whether we should add these WEBPACK_* together to align with webpack-cli. I didn't do it as it will break those who use WEBPACK_* to determine who (webpack or rspack) is using the function configuration .

I would say let's hold the `WEBPACK_*` for now.

Reply:
> > another question is whether we should add these WEBPACK_* together to align with webpack-cli. I didn't do it as it will break those who use WEBPACK_* to determine who (webpack or rspack) is using the function configuration .
> 
> I would say let's hold the `WEBPACK_*` for now.

ok